.travis.yml

```diff
@@ -40,7 +40,11 @@ before_install:
     - pip install vtk
     - pip install mayavi
     - mkdir -p $SUBJECTS_DIR
-    - pip install "https://api.github.com/repos/mne-tools/mne-python/zipball/master"
+    - if [ "${PYTHON_VERSION}" == "3.5" ]; then
```

Comment:
We could drop 3.5 support? Most pydata packages have...

Reply:
Let's do it in another PR. It would be good to do that while also adding 3.8 now that VTK9 is out. But that might require quite a few iterations...